tasks/waf/install/install-alpine.yml

```diff
@@ -1,29 +1,32 @@
 ---
-- name: (Alpine Linux) {{ nginx_license_status is defined | ternary('Remove', 'Configure') }} NGINX Plus repository
-  lineinfile:
-    path: /etc/apk/repositories
-    insertafter: EOF
-    line: "{{ nginx_plus_repository | default(nginx_plus_default_repository_alpine) }}"
-    state: "{{ nginx_license_status | default ('present') }}"
-  when: nginx_app_protect_manage_repo | bool
+- name: (Alpine Linux) Create Repositories
```

Comment:
Minor nitpick but this name can be misleading if you then proceed to remove the NGINX Plus repository